Makefile.am

```diff
@@ -116,7 +116,8 @@ setup.h: $(srcdir)/setup.h.in Makefile
 	     -e 's|@prefix[@]|$(prefix)|g' \
 	     -e 's|@p4includedir[@]|$(p4includedir)|g' \
 	     -e 's|@p4_14includedir[@]|$(p4_14includedir)|g' \
-	$< >$@
+	$< >$@.tmp
+	@if cmp $@ $@.tmp; then rm $@.tmp; else mv $@.tmp $@; fi
```

Comment:
This is an attempt to not rebuild everything all the time -- all object files depend on setup.h, and setup.h depends on Makefile, so by not updating setup.h when there are no actual differences, we don't need to rebuild everything whenever Makefile changes.  Unfortunately there still seem to be dependencies on Makefile.am somewhere that rebuild everything when Makefile.am changes.